### DISCUSSION TITLE
Github actions workflow trigger

### DISCUSSION THREAD

**Discussion by Shalom-Nankam** [community/community#40258] [Actions]
### Select Topic Area

General

### Body

Github actions workflow makes use of yaml type documents. Now, in the yaml language , the word 'on' used as the trigger key in actions workflow is a sort of keyword that means the same as the boolean 'true' in majority other languages.

In trying to apply a bit of automation to my actions workflow by dynamically changing some values(environment variables), editing the workflow document by converting the yaml to another type(dict in python) and back to yaml would ALWAYS change the 'on' to 'true' thereby causing issues in my workflow doc.

Please what would be a workaround for this?

**Comment by ossanna16** [comment]
Hi there @Shalom-Nankam and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by Shalom-Nankam** [comment]
Issue Resolved. It had to do with my parsing tool still making use of the YAML 1.1 specs. The YAML 1.2 spec does not give this issue and a parser that uses this spec will work
